yaml2ncml/yaml2ncml.py

```diff
@@ -29,61 +32,16 @@
 """
 
 
-def str_att(name, value):
+def attribute_name(name, value):
     if isinstance(value, list):
         value = ','.join(value)
     msg = '  <attribute name="{:s}" type="String" value="{:s}"/>\n'
     return msg.format(name, value)
 
 
-def header():
-    text = '<?xml version="1.0" encoding="UTF-8"?>\n<netcdf xmlns='
-    text += '"http://www.unidata.ucar.edu/namespaces/netcdf/ncml-2.2">\n'
-    text += str_att('Conventions', 'CF-1.6, SGRID-0.1, ACDD-1.3')
-    text += str_att('cdm_data_type', 'Grid')
-    return text
-
-
-def footer(text):
-    text += '</netcdf>\n'
-    return text
-
-
-def add_global_atts(text, a):
-    d = a['dataset']
-    for key, value in d.items():
-        # Handle simple attribute pairs first.
-        if key in ['id', 'license', 'summary', 'title', 'project',
-                   'naming_authority', 'references', 'acknowledgments']:
-            text += str_att(key, value)
-        elif key in ['creator', 'publisher']:
-            email = value.get("email", None)
-            if email:
-                text += str_att('_'.join([key, 'email']), email)
-            url = value.get("url", None)
-            if url:
-                text += str_att('_'.join([key, 'url']), url)
-            name = value.get("name", None)
-            if name:
-                text += str_att('_'.join([key, 'name']), name)
-        elif key in ['contributor']:
-            role = value.get("role", None)
-            text += str_att('_'.join([key, 'role']), role)
-            email = value.get("email", None)
-            if email:
-                text += str_att('_'.join([key, 'email']), email)
-            url = value.get("url", None)
-            if url:
-                text += str_att('_'.join([key, 'url']), url)
-            name = value.get("name", None)
-            if name:
-                text += str_att('_'.join([key, 'name']), name)
-    return text
```

Comment:
`header` and `add_global_atts` are parsed with the template. The renaming fields will be added later.